.rubocop.yml

```diff
@@ -1,6 +1,6 @@
 AllCops:
   NewCops: enable
-  TargetRubyVersion: 2.5.1
+  TargetRubyVersion: 3.0.0
```

Comment:
I'd add a different commit for this change since it doesn't have anything to do with this PR, what do you think?